doc/AUTHORS.adoc

```diff
@@ -7,32 +7,37 @@ maintained by Joel Rosdahl.
 ccache is a collective work with contributions from many people, including:
 
 * Alexander Korsunsky
+* Alexander Lanin
 * Alexey Tourbin
 * Alfred Landrum
 * Anders F Björklund
+* André Klitzing
 * Andrea Bittau
```

Comment:
Looks like you have sorted using the C locale instead of e.g. en_US? I think you can revert the changes to this file; they are done before a release.

Reply:
This was mostly to verify update_authors is working. It's still producing slightly different results, e.g. Tomasz before Tomm. I was assuming -d fixes that, but it does not. Any ideas?

Reply:
I've added this file simply to verify update_authors works as expected (no unexpected diff in the file).